src/System.Private.CoreLib/src/System/Reflection/Emit/CustomAttributeBuilder.cs

```diff
@@ -394,35 +394,35 @@ private static void EmitString(BinaryWriter writer, string str)
             writer.Write(utf8Str);
         }
 
-        private static void EmitValue(BinaryWriter writer, Type type, object value)
+        private static void EmitValue(BinaryWriter writer, Type type, object? value)
         {
             if (type.IsEnum)
             {
                 switch (Type.GetTypeCode(Enum.GetUnderlyingType(type)))
                 {
                     case TypeCode.SByte:
-                        writer.Write((sbyte)value);
+                        writer.Write((sbyte)value!); // TODO-NULLABLE https://github.com/dotnet/roslyn/issues/34976
```

Comment:
> // TODO-NULLABLE https://github.com/dotnet/roslyn/issues/34976 [](start = 52, length = 63)

This issue isn't relevant here.  The case described in that issue is taking some value type and casting it through object back to that value type; in that case the compiler could see that the object would not be null.  Here, though, you've just got an `object?`; there's no way the compiler can see whether it's definitively non-null.

Reply:
removed this and similar ones